Doc/library/io.rst

```diff
@@ -770,6 +777,11 @@ than raw I/O does.
       Read and return *size* bytes, or if *size* is not given or negative, until
       EOF or if the read call would block in non-blocking mode.
 
+      .. note::
+         When the underlying raw stream is non-blocking, a ``BlockingIOError``
+         may be raised if the read operation cannot be completed immediately.
+         Ensure proper exception handling in such cases.
```

Comment:
Personal nitpick / my individual style leanings: I don't think the "Ensure proper exception handling in such cases" part is needed. Definitely not a python doc style expert though / leave to others.

Reply:
check.

Reply:
check